discovery-provider/src/api/v1/tracks.py

```diff
@@ -361,6 +361,26 @@ def get(self):
 
 # Stream
 
+stream_parser = reqparse.RequestParser(argument_class=DescriptiveArgument)
+stream_parser.add_argument(
+    "user_signature",
+    description="""Optional - signature from the requesting user's wallet.
+        This is needed to authenticate the user and verify access in case the track is premium.""",
+    type=str,
+)
+stream_parser.add_argument(
+    "user_data",
+    description="""Optional - data which was used to generate the optional signature argument.""",
+    type=str,
+)
+stream_parser.add_argument(
+    "premium_content_signature",
+    description="""Optional - premium content signature for this track which was previously generated by a registered DN.
```

Comment:
hm no action needed on this, but fyi the ts generator didn't keep the newlines here unf.